### ISSUE TITLE
Not able to automatically join a private group when it has been created by another client while Dino was open at the same time

### ISSUE THREAD

**Issue by KillianKemps** [dino/dino#810]
When I create a private group through Conversations it gives a default name composed with the list of the members.

When I access this private group with Dino, I do not have the icon to encrypt the messages with OMEMO. I can send a message but it is sent unencrypted.

To circumvent this I found I had to give a name to this private group (through Conversations or through Dino), then I reopen the chat and finally I see the lock icon in the button and I can switch it to OMEMO encryption.

Could you make Dino encrypt the messages with OMEMO even when there is no name to the private group?

Dino version : 0.1.0

**Comment by licaon-kter** [comment]
I might think the actual close/open action fixed it, as this was a bug at some point.

**Comment by KillianKemps** [comment]
Yes you are right. I tried again and the bug is different from what I first described.

In fact, if I create a private group with Conversations while Dino is already open, it doesn't detect the conversation automatically.
I have to manually join the channel through Dino to be able to see it. And then I have this issue that I can only send unencrypted messages without being able to switch to encrypted messages with the lock icon.

However, if I create a private group with Conversations and Dino is already open. I don't need to try to join the group manually. I can just close and reopen Dino. Then the group is automatically joined and I can switch the lock icon to choose to encrypt the messages.

So, I see two bugs:
1. Dino is not able to automatically join a private group when it has been created by Conversations and Dino is open at the same time. We need to close and reopen Dino to circumvent.
2. Dino doesn't choose OMEMO encryption automatically in private MUC while others are using it.
